src/Symfony/Component/Config/Exception/FileLoaderLoadException.php

```diff
@@ -11,99 +11,15 @@
-     * @param string     $sourceResource The original resource importing the new resource
-     * @param int        $code           The error code
-     * @param \Exception $previous       A previous exception
-     * @param string     $type           The type of resource
-     */
-    public function __construct(string $resource, string $sourceResource = null, int $code = null, \Exception $previous = null, string $type = null)
-    {
-        $message = '';
-        if ($previous) {
-            // Include the previous exception, to help the user see what might be the underlying cause
+class_alias(LoaderLoadException::class, FileLoaderLoadException::class);
 
-            // Trim the trailing period of the previous message. We only want 1 period remove so no rtrim...
-            if ('.' === substr($previous->getMessage(), -1)) {
-                $trimmedMessage = substr($previous->getMessage(), 0, -1);
-                $message .= sprintf('%s', $trimmedMessage).' in ';
-            } else {
-                $message .= sprintf('%s', $previous->getMessage()).' in ';
-            }
-            $message .= $resource.' ';
-
-            // show tweaked trace to complete the human readable sentence
-            if (null === $sourceResource) {
-                $message .= sprintf('(which is loaded in resource "%s")', $this->varToString($resource));
-            } else {
-                $message .= sprintf('(which is being imported from "%s")', $this->varToString($sourceResource));
-            }
-            $message .= '.';
-
-        // if there's no previous message, present it the default way
-        } elseif (null === $sourceResource) {
-            $message .= sprintf('Cannot load resource "%s".', $this->varToString($resource));
-        } else {
-            $message .= sprintf('Cannot import resource "%s" from "%s".', $this->varToString($resource), $this->varToString($sourceResource));
-        }
-
-        // Is the resource located inside a bundle?
-        if ('@' === $resource[0]) {
-            $parts = explode(DIRECTORY_SEPARATOR, $resource);
-            $bundle = substr($parts[0], 1);
-            $message .= sprintf(' Make sure the "%s" bundle is correctly registered and loaded in the application kernel class.', $bundle);
-            $message .= sprintf(' If the bundle is registered, make sure the bundle path "%s" is not empty.', $resource);
-        } elseif (null !== $type) {
-            // maybe there is no loader for this specific type
-            if ('annotation' === $type) {
-                $message .= ' Make sure annotations are installed and enabled.';
-            } else {
-                $message .= sprintf(' Make sure there is a loader supporting the "%s" type.', $type);
-            }
-        }
-
-        parent::__construct($message, $code, $previous);
-    }
+@trigger_error('The FileLoaderLoadException class is deprecated since Symfony 4.2, use LoaderLoadException instead', E_USER_DEPRECATED);
```

Comment:
sorry I was too laconic in my previous messages: we should to the same as in other places: put the FQCN, and double quotes around the class name:
`@trigger_error(sprintf('The "%s" class is deprecated since Symfony 4.2, use "%s" instead.', FileLoaderLoadException::class, LoaderLoadException::class), E_USER_DEPRECATED);`

Reply:
Oh, ok.